Jenkinsfile

```diff
@@ -106,6 +107,22 @@ def publishChart(imageName) {
     }
   }
 }
+
+def triggerDeploy(jenkinsUrl, jobName, token, params) {
+  // jenkinsUrl is the full url (without trailing /) of the jenkins service, including username and access token
+  // e.g. https://deploy:accesstoken@jenkins.ffc.aws-int.defra.cloud
+  // jobName is the jenkins job name, this is in the url when you edit or view the job in Jenkins
+  // token is the token that is set up when you configured the job in Jenkins. You must tick the "Trigger builds remotely" option when configuring the job. The Authentication token entered
+  //    into the job is the one that should be passed here.
+  // params is an object, that should contain all the parameters that need to be passed to the job (if required), for example ['version': '1.0.0']
+  def url = "$jenkinsUrl/job/$jobName/buildWithParameters?token=$token"
+  params.each { param ->
+    url = url + "\\&amp;$param.key=$param.value"
+  }
+  println "Triggering deployment for $url"
```

Comment:
The convention seems to be to use `echo` rather than `println` in the Jenkins scripts I've seen

Reply:
I've changed it to use echo in the shared library

Reply:
I've also moved the token into a credential, one per service and configured those in Jenkins